### PR TITLE
locust: Migrate to new transaction API

### PR DESCRIPTION
**Pull request by akashin**
We migrate away from `broadcast_tx_commit` and `broadcast_tx_async` which are now deprecated in favor of `send_tx`: https://docs.near.org/api/rpc/transactions

I believe this PR should preserve the behavior that we had before and I plan to do any behavior changes in the future PRs.

Tested by running:
```sh
locust -H 127.0.0.1:3030 \
  -f locustfiles/ft.py \
  --funding-key=$KEY --headless -u 2000 -r 50 -t 120s
```

This is a part of work on improving the performance of Locust load test runner: https://github.com/near/nearcore/issues/11352

### CODE REVIEW

**Comment by akashin** [comment]
Actually, this has a behavior change: `send_tx` does not return transaction id, but `broadcast_tx_async` does, so I need to figure out how to backfill that behavior.

File: pytest/tests/loadtest/locust/common/base.py [hunk 1]
```diff
@@ -277,16 +277,24 @@ def send_tx(self, tx: Transaction, locust_name: str) -> dict:
             try:
                 # To get proper errors on invalid transaction, we need to use sync api first
```
Comment (Longarithm) [inline pytest/tests/loadtest/locust/common/base.py:278]:
```suggestion
                # To get proper errors on invalid transaction, we need to wait at least for optimistic execution
```
maybe that's more precise?

**Review by Longarithm — APPROVED** [review]
From what I know about new API - makes sense.

**Comment by Longarithm** [comment]
> Actually, this has a behavior change: send_tx does not return transaction id, but broadcast_tx_async does, so I need to figure out how to backfill that behavior.

But I thought tx id is uniquely determined by user even before sending `SignedTransaction`?
cc @telezhnaya

File: pytest/tests/loadtest/locust/common/base.py (hunk 1 above)
Comment (akashin) [reply pytest/tests/loadtest/locust/common/base.py:278]: Thanks, accepted!

**Pull request merged**